version/gradle.properties

```diff
@@ -1,11 +1,11 @@
-#Thu, 20 Apr 2017 11:30:12 -0700
+#13:17 21/04/2017
 version_minecraft=1.10.2
 version_forge=12.18.3.2239
 version_mcp=snapshot_20160826
 version_forestry=[5.1.15.295,)
-version_industrialcraft=[2.6.11-ex110,2.7.0-ex111)
+version_industrialcraft=2.6.188-ex110
 version_project_major=10
 version_project_series=1
 version_project_revision=0
-version_project_build=0
-version_project_qualifier=none
+version_project_build=3.1
+version_project_qualifier=beta
```

Comment:
Alright I'll revert them, I wasn't really too sure about what they were supposed to be.